tests/toolkit/test_utils.py

```diff
@@ -0,0 +1,56 @@
+from goose.toolkit.utils import parse_plan
+
+
+def test_parse_plan_simple():
+    plan_str = (
+        "Here is python repo\n"
+        "-use uv\n"
+        "-do not use poetry\n\n"
+        "Now you should:\n\n"
+        "-Open a file\n"
+        "-Run a test"
+    )
+    expected_result = {
+        "kickoff_message": "Here is python repo\n-use uv\n-do not use poetry\n\nNow you should:",
+        "tasks": ["-Open a file", "-Run a test"],
+    }
+    assert expected_result == parse_plan(plan_str)
+
+
+def test_parse_plan_multiple_groups():
+    plan_str = (
+        "Here is python repo\n"
+        "-use uv\n"
+        "-do not use poetry\n\n"
+        "Now you should:\n\n"
+        "-Open a file\n"
+        "-Run a test\n\n"
+        "Now actually follow the steps:\n"
+        "-Step1\n"
+        "-Step2"
+    )
+    expected_result = {
+        "kickoff_message": (
+            "Here is python repo\n"
+            "-use uv\n"
+            "-do not use poetry\n\n"
+            "Now you should:\n\n"
+            "-Open a file\n"
+            "-Run a test\n\n"
+            "Now actually follow the steps:"
+        ),
+        "tasks": ["-Step1", "-Step2"],
```

Comment:
[nit/maybe not even applicable]should we be stripping ` - ` from the tasks? Just wondering because if someone happens to type in a plan:
```
-1:do this 
-2:do that
```
then that _could_ be inferred as negative numbers which _maybe_ would change the order of events?

Reply:
agree!